src/FrameBase.re

```diff
@@ -280,10 +280,14 @@ external selectAllEvalPromise4:
   "$$eval";
 
 [@bs.send]
-external addScriptTag: (t, tagOptions) => Js.Promise.t(ElementHandle.t) = "";
+external addScriptTag:
+  (t, tagOptions) => Js.Promise.t(ElementHandle.t(Dom.element)) =
```

Comment:
I don't know of any bindings to script tag elements specifically (none in `bs-webapi` at least). I think this is probably OK for now, as I sort of doubt this return type is going to be used much. The function is primarily used for the side effect.